.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,90 @@
+name: "Bug report"
+description: "Create a report to help us improve TypeScript"
+body:
+  - type: markdown
+    attributes:
+      value: Please fill in each section completely. Thank you!
+  - type: input
+    id: search_terms
+    attributes:
+      label: "🔎 Search Terms"
+      description: |
+        What search terms did you use when trying to find an existing bug report?
+
+        List them here so people in the future can find this one more easily.
+    validations:
+      required: true
+  - type: textarea
+    id: version_info
+    attributes:
+      label: "🕗 Version & Regression Information"
+      description: |
+        When did you start seeing this bug occur?
+
+        \"Bugs\" that have existed in TS for a long time are very likely to be FAQs; refer to
+        https://github.com/Microsoft/TypeScript/wiki/FAQ#common-bugs-that-arent-bugs
+
+
+        If possible, please try testing the nightly version of TS to see if it's already been fixed. For npm: `typescript@next`
+
+        This is also the 'Nightly' version in the playground: http://www.typescriptlang.org/play/?ts=Nightly
+  
+
+        Note: The TypeScript Playground can be used to try older versions of TypeScript.
+
+        Please keep and fill in the line that best applies.
+      value: |
+        - This is a crash
+        - This changed between versions ______ and _______
+        - This is the behavior in every version I tried, and I reviewed the FAQ for entries about _________
+        - I was unable to test this on prior versions because _______
+      render: Markdown
+    validations:
+      required: true
+  - type: input
+    id: playground_link
+    attributes:
+      label: ⏯ Playground Link
+      description: |
+        A link to a TypeScript Playground \"Share\" link which shows this behavior.
+
+        This should have the same code as the code snippet below, and use whichever settings are relevant to your report.
+
+
+        As a last resort, you can link to a repo, but these will be slower for us to investigate.
+      placeholder: "Playground link with relevant code: https://www.typescriptlang.org/play?#code/PTAEFkE9QYwewCYFNQHM5IM6gBZIE5JA"
+    validations:
+      required: false
+  - type: textarea
+    id: code
+    attributes:
+      label: 💻 Code
+      description: |
+        Please post the relevant code sample here as well.
+        This code and the Playground code should be the same, do not use separate examples.
+      placeholder: |
```

Comment:
Testing this one out in a real form, a placeholder doesn't look like it works here. The text shows up, but since it's just a placeholder, writing anything means that this goes away.

What this should be is part of the description.

https://github.com/jakebailey/ts-issue-template-testing/issues/new?assignees=&labels=&projects=&template=bug_report.yml

Reply:
If we use `value`, the text will be kept, thus allowing the user to modify it. I'll use that.

Reply:
But this isn't text that needs to be updated; it's a list of requirements/suggestions for what the box will contain. I don't think there's any reason for it to be a part of the input itself.

Reply:
Hmm, alright, sorry for the confusion 😅 Will add it to the description.